ldms/src/sampler/Makefile.am

```diff
@@ -302,6 +302,13 @@ if ENABLE_BLOB_STREAM
 SUBDIRS += blob_stream
 endif
 
+if ENABLE_ZFS
+SUBDIRS += zfs_topvdevs
+SUBDIRS += zfs_leafvdevs
+SUBDIRS += zfs_zpool
+EXTRA_DIST = zfs_common.h
```

Comment:
The EXTRA_DIST should be _outside_ of the conditional. The "distribution" always needs to include all of the source files. At distribution time, configuration decisions have not yet been made yet.